buildSrc/build.gradle.kts

```diff
@@ -11,7 +11,7 @@ repositories {
 }
 
 dependencies {
-  implementation("com.android.tools.build:gradle:3.5.3")
+  implementation("com.android.tools.build:gradle:3.6.0")
```

Comment:
I am probably going to make a PR upgrading the build tools now as the AS 3.6 update dropped just as I was done work yesterday

Reply:
OK. I'll revert that.